api/src/main/java/jakarta/persistence/EntityAgent.java

```diff
@@ -0,0 +1,266 @@
+/*
+ * Copyright (c) 2008, 2025 Oracle and/or its affiliates. All rights reserved.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0,
+ * or the Eclipse Distribution License v. 1.0 which is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause
+ */
+
+// Contributors:
+//     Gavin King      - 4.0
+
+package jakarta.persistence;
+
+import java.util.List;
+
+/**
+ * Interface enabling more direct control over interaction with the
+ * database than what is possible with {@link EntityManager}.
```

Comment:
I tried to think of a summary that will give the user an overview of what this interface gives them,
```suggestion
 * Provides entity operations that are performed independently
 * of a persistence context. This interface enables more direct
 * control over interaction with the database than what is
 * possible with {@link EntityManager}.
```